docs/deploy-rbd.md

```diff
@@ -252,7 +252,7 @@ be able to review jwt tokens.
 Configure a role(s) for service accounts used for ceph-csi:
 
 * provisioner service account (`rbd-csi-provisioner`) requires only **delete**
-  permissions to delete passphrases on pvc delete
+  permissions to delete passphrases on PVC delete
```

Comment:
can you also update expand-pvc.md and snapshot.md

Reply:
Oh.. sure.